k8s/object.bzl

```diff
@@ -65,7 +63,7 @@ def _impl(ctx):
     # to include as runfiles, so they are accessible to be pushed.
     for tag in ctx.attr.images:
       target = ctx.attr.images[tag]
-      image = _get_layers(ctx, image_target_dict[target], image_files_dict[target])
+      image = _get_layers(ctx, image_target_dict[target])
```

Comment:
FYI, the signature changed again - now it's `(ctx, name, attr_target)` ([more here](https://github.com/bazelbuild/rules_k8s/issues/125#issuecomment-395416493)).